shapash/explainer/smart_plotter.py

```diff
@@ -452,15 +461,17 @@ def plot_violin(self,
 
             else:
                 fig.add_trace(go.Violin(x=feature_values.loc[feature_values.iloc[:, 0] == i].values.flatten(),
-                                        y=contributions.loc[feature_values.iloc[:, 0] == i].values.flatten(),
+                                        y=contributions.loc[feature_values.iloc[:, 0]
+                                                            == i].values.flatten(),
                                         line_color=self.default_color,
                                         showlegend=False,
                                         meanline_visible=True,
                                         scalemode='count',
-                                        hovertext=hv_text_df.loc[feature_values.iloc[:, 0] == i].values.flatten(),
-                                        hovertemplate='<b>%{hovertext}</b><br />' + hv_temp,
-                                        customdata=contributions.index.values
-                                        ))
+                                        hovertext=hv_text_df.loc[feature_values.iloc[:, 0] == i].values.flatten(
```

Comment:
code become difficult to read with flatten(
),

Reply:
It's not coming from myside

Reply:
generate a loss of clarity